Beta behaviour

mattagape opened the pull request:
Mainly to ensure Grafana and Postgres both update users correctly.

Commit:
Adding code to update Grafana users.

Commit:
Now we can update Grafana user password.

Commit:
Removed duplicate files.

Commit:
Replace 2 similar functions with 1.

Commit:
Merge branch 'beta' into beta-behaviour

Commit:
A correction and a clarification.

Commit:
Fix permissions error.

Commit:
Updated poetry file.

Llannelongue commented:
Quick question on this PR: are the changes to the following files related to adding users across the DB and Grafana?
- `setup_environment.sh`
- and the files under `tests/`

mattagape commented:
> Quick question on this PR: are the changes to the following files related to adding users across the DB and Grafana?
> 
> * `setup_environment.sh`
> * and the files under `tests/`

Good catch. I'm afraid not. The files under `/tests` had to be changed so that the unit tests would work after I removed the duplicate files from there. The environment script I added a single extra line as it kept failing before I did so.

Llannelongue commented:
Ok, then can you please amend the PR to only adress this PR's topic? (or delete it and start a new PR)

Llannelongue reviewed (changes requested):
Remove changes not related to the PR's topic

Commit:
Revert "Removed duplicate files."

This reverts commit 223b2db596bd878befeba03f9abd8189820b2aee.

Commit:
Revert "Replace 2 similar functions with 1."

This reverts commit f8520a5bd97cd46568900b81fa81d3a251fbf409.

Commit:
Revert "Fix permissions error."

This reverts commit f07ec8f21a442617c6f0b5724b53fb66a2dc5a6b.

Commit:
Reverting changes.

mattagape commented:
Reverted the changes so they can be done on another PR.

Llannelongue commented:
Closing at this stage since the code doesn't run as is.
